pkg/common/utils/gang_TGInstanceCountMap.go

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
this file name is too long. Since we have a short term "tg" for task group, I suggest renaming this file to:  `tg_map.go`